.ci/pytorch/test.sh

```diff
@@ -400,7 +400,15 @@ pr_time_benchmarks() {
   source benchmarks/dynamo/pr_time_benchmarks/benchmark_runner.sh "$TEST_REPORTS_DIR/pr_time_benchmarks_after.txt" "benchmarks/dynamo/pr_time_benchmarks/benchmarks"
   echo "benchmark results on current PR: "
   cat  "$TEST_REPORTS_DIR/pr_time_benchmarks_after.txt"
-
+  git checkout HEAD~1
+
+  # we can try firt to run develop alone, and if it worked then we do not need to run clean->develop.
+  # another suggestion was to use historical stored data insetad of building Head~1
+  python setup.py clean
+  python setup.py develop
```

Comment:
Maybe
```
python setup.py develop --cmake
```
which should be good enough for 99% of changes and is a lot faster than clean.

Reply:
and we fail for the remaining 1%? i will try to mimic test_forward_backward_compatibility